README.md

```diff
@@ -18,28 +18,34 @@
     var express = require('express');
     var app = express();
 
-    // create the view engine with `react-engine`
-    var engine = require('react-engine').server.create({
-      reactRoutes: <string> /* pass in the path to react-router routes optionally */
-      performanceCollector: <function> /* optional function to collect perf stats */
-    });
+    app.use(function(req, res) {
```

Comment:
@samsel No problem! It allows you to handle the onAbort method and use res.redirect. That gets called when you do a transition.redirect. Say you have a high order component called AuthComponent that checks the SessionStore for an active session and if not it redirects you to the /signup route. If on abort isn't handled it will throw an error.